Source/CarthageKitTests/XcodeSpec.swift

```diff
@@ -36,6 +34,7 @@ class XcodeSpec: QuickSpec {
 		}
 
 		describe("determingSwiftInformation") {
+			let currentSwiftVersion = swiftVersion.single()?.value
```

Comment:
Ah, I missed that. Not sure how I feel about that approach, but it should be good enough. 👍